javascript/src/tracing/setup.ts

```
@@ -1,5 +1,7 @@
+import { OpenTelemetry } from "@ai-sdk/otel";
```

ESLint

[error] Unable to resolve path to module '`@ai-sdk/otel`'.

(import/no-unresolved)